docs/rules/valid-v-slot.md

```diff
@@ -0,0 +1,112 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/valid-v-slot
+description: enforce valid `v-slot` directives
+---
+# vue/valid-v-slot
+> enforce valid `v-slot` directives
+
+This rule checks whether every `v-slot` directive is valid.
+
+## :book: Rule Details
+
+This rule reports `v-slot` directives in the following cases:
+
+- The directive is not owned by a custom element. E.g. `<div v-slot=""></div>`
+- The directive is a named slot and is on a custom element directly. E.g. `<my-component v-slot:foo></my-component>`
+- The directive is the default slot, is on a custom element directly, and there are other named slots. E.g. `<my-component v-slot=""><template v-slot:foo></template></my-component>`
+- The element which has the directive has another `v-slot` directive. E.g. `<my-component v-slot:one v-slot:two></my-component>`
+- The element which has the directive has another `v-slot` directive that is distributed to the same slot. E.g. `<my-component><template v-slot:foo></template><template v-slot:foo></template></my-component>`
+- The directive has a dynamic argument which uses the scope properties that the directive defined. E.g. `<my-component><template v-slot:[data]="data"></template></my-component>`
+- The directive has any modifier. E.g. `<my-component v-slot.foo></my-component>`
+- The directive is the default slot, is on a custom element directly, and has no value. E.g. `<my-component v-slot></my-component>`
+
+<eslint-code-block :rules="{'vue/valid-v-slot': ['error']}">
+
+```vue
+<template>
+  <!-- ✓ GOOD -->
+  <my-component v-slot="data">
+    {{data}}
+  </my-component>
+  <my-component>
+    <template v-slot:default>
+      default
+    <template>
+    <template v-slot:one>
+      one
+    <template>
```

Comment:
I'm sorry. I overlooked it. The two above it were not closed as well.

Reply:
Oops, thank you!